CMakeLists.txt

```diff
@@ -184,6 +184,31 @@ elseif(NMC_VECTORIZE_TARGET STREQUAL "AVX2")
 endif()
 
 #----------------------------------------------------------
+# Which communicator algorithm to use for finding
+#  spike:connection matches: linear/global/domain
+#  communication is O(nodes)
+#  spike  is O(log(nodes))
+#  global is O(log(nodes)),
+#  domain is O(log(neurons/nodes))
+# if the number of spikes goes up with number of nodes
+#----------------------------------------------------------
+set(NMC_USE_QUEUE "domain" CACHE STRING
+  "set the communication connection algorithm, one of connection/spike/global/domain")
+set_property(CACHE NMC_USE_QUEUE PROPERTY STRINGS connection spike global domain)
```

Comment:
I thought that's what I was doing here... It looks to me like what you're referencing.